fabric-data-generation-api-v1/src/main/java/net/fabricmc/fabric/api/datagen/v1/provider/FabricTagProvider.java

```diff
@@ -210,10 +210,10 @@ protected DynamicRegistryTagProvider(FabricDataGenerator dataGenerator, Registry
 	/**
 	 * An extension to {@link net.minecraft.data.server.AbstractTagProvider.ObjectBuilder} that provides additional functionality.
 	 */
-	public class FabricTagBuilder<O> extends ObjectBuilder<O> {
-		private final AbstractTagProvider.ObjectBuilder<O> parent;
+	public class FabricTagBuilder<T> extends ObjectBuilder<T> {
```

Comment:
If the self-returning methods are final, better just declare this class final as well (as otherwise subclasses cannot return a more reified self type)

Reply:
Made the class final, some methods still have explicit final because of `@SafeVarargs`.